cirq/google/engine/engine.py

```diff
@@ -276,6 +277,74 @@ def run_sweep(
                                         description=job_description,
                                         labels=job_labels)
 
+    def run_batch(
+            self,
+            programs: List['cirq.Circuit'],
+            program_id: Optional[str] = None,
+            job_id: Optional[str] = None,
+            params_list: List[study.Sweepable] = None,
+            repetitions: int = 1,
+            processor_ids: Sequence[str] = ('xmonsim',),
```

Comment:
suggest not having a default and yelling if not supplied.  this is different from other cases, but there is an issue out to not have this default (it is too dangerous, you could think you were doing an experiment, but actually be running on a simulator).

Reply:
Done.